.github/workflows/publish.yml

```
@@ -20,7 +20,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v1
         with:
-          python-version: 3.9
+          python-version: 3.10
```

I think we'll need the quotes here too, so YAML doesn't parse this as the float `3.1`.

```suggestion
          python-version: "3.10"
```